src/maintenance/performance.rst

```diff
@@ -212,6 +212,21 @@ This TCP buffering behaviour can be disabled via
 .. seealso::
     Bulk :ref:`load <api/db/all_docs>` and :ref:`store <api/db/bulk_docs>` API.
 
+Connection limit
+----------------
+
+`MochiWeb`_ is handling CouchDB requests.
+By default maximum number of connections is 2048. To change this limit, use
+server_options configuration variable. 'max' indicates maximum number of
+connections.
+
+.. code-block:: ini
+
+    [httpd]
```

Comment:
For CouchDB 2.x

I think this needs to be `[chttpd]` to be applied to the main (clustered) port (by default 5984).

`[httpd]` would be for the backend (node local) port (by default 5986).


For CouchDB 1.x `[httpd]` works. But I think that should be a separate PR against the 1.6.x branch.